packages/pigeon/test/pigeon_test.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:pigeon/pigeon.dart';
```

Comment:
This file needs a license header, as indicated by the failing CI step.

Reply:
Added the same licence as other files